Packs/CommonTypes/IndicatorFields/indicatorfield-Report_type.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "indicator_reporttype",
+    "version": -1,
+    "modified": "2023-03-05T10:02:22.503613368Z",
+    "name": "Report type",
+    "ownerOnly": false,
+    "cliName": "reporttype",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
```

Comment:
why is that` associatedToAll : tru`e and user ID field `associatedToAll : false`?

Reply:
switched it to `True`